src/status_im/common/alert_banner/view.cljs

```diff
@@ -0,0 +1,57 @@
+(ns status-im.common.alert-banner.view
+  (:require [quo.core :as quo]
+            [quo.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [react-native.hole-view :as hole-view]
+            [react-native.safe-area :as safe-area]
+            [status-im.common.alert-banner.style :as style]
+            [utils.re-frame :as rf]))
+
+(def colors-map
+  {:alert
+   {:background-color colors/warning-50-opa-20
+    :text-color       colors/warning-60}
+   :error
+   {:background-color colors/danger-50-opa-20
+    :text-color       colors/danger-60}})
+
+(defn- banner
+  [{:keys [text type]} second-banner?]
+  [rn/view (when second-banner? {:style style/second-banner-wrapper})
+   [hole-view/hole-view
+    {:style (style/hole-view (get-in colors-map [type :background-color]))
+     :holes (if second-banner?
+              []
+              [{:x            0
+                :y            40
+                :width        (:width (rn/get-window))
+                :height       40
+                :borderRadius 20}])}
+    [quo/text
+     {:size   :paragraph-2
+      :weight :medium
+      :style  {:color (get-in colors-map [type :text-color])}}
+     text]]])
+
+(defn view
+  []
+  (let [banners       (rf/sub [:alert-banners])
+        banners-count (count banners)
+        alert-banner  (:alert banners)
+        error-banner  (:error banners)
+        safe-area-top (safe-area/get-top)]
+    [hole-view/hole-view
+     {:style {:background-color colors/neutral-100}
+      :holes [{:x            0
+               :y            (+ safe-area-top (* 40 banners-count))
+               :width        (:width (rn/get-window))
+               :height       40
```

Comment:
We could move `40` to a constant. Later, if the banner height is dynamic, we can change it with a sub.